src/Microsoft.ML.FastTree/GamTrainer.cs

```diff
@@ -1285,8 +1285,9 @@ public Context(IChannel ch, GamModelParametersBase pred, RoleMappedData data, IE
                             new RoleMappedSchema.ColumnRole(MetadataUtils.Const.ScoreValueKind.Score).Bind(DefaultColumnNames.Score));
                     }
 
-                    _data.Schema.Schema.TryGetColumnIndex(DefaultColumnNames.Features, out int featureIndex);
-                    MetadataUtils.TryGetCategoricalFeatureIndices(_data.Schema.Schema, featureIndex, out _catsMap);
+                    var featureCol = _data.Schema.Schema.GetColumnOrNull(DefaultColumnNames.Features);
+                    ch.Assert(featureCol.HasValue);
+                    MetadataUtils.TryGetCategoricalFeatureIndices(_data.Schema.Schema, featureCol.Value.Index, out _catsMap);
```

Comment:
In this particular case, you are asserting that it has a value, which is fine, but if the purpose is to just to clarify this, you could *probably* just get away with just doing `.Value` at the end and operating over that, since that also clarifies, "yes, I definitely expect this to have a value."